tools/lkl/lib/hijack/hijack.c

```diff
@@ -161,7 +161,17 @@ HOOK_FD_CALL(vmsplice)
 HOOK_CALL_USE_HOST_BEFORE_START(pipe);
 
 HOOK_CALL_USE_HOST_BEFORE_START(accept4);
+int accept4(int fd, struct sockaddr *addr, socklen_t *addrlen, int flags)
+{
+	return lkl_call(__lkl__NR_accept4, 4, fd, addr, addrlen, flags);
+}
+
+
 HOOK_CALL_USE_HOST_BEFORE_START(pipe2);
+int pipe2(int pipefd[2], int flags)
+{
```

Comment:
Same, here, it has been a long time, could you please remember why do we need this?